docs/deploying.md

```diff
@@ -54,7 +54,25 @@ If the specified config file does not yet exist, you will again be prompted to c
 
 ## Automated deploys
 
-To set up automatic deploys (also known as *continuous deployment* or *CD*), we recommend [GitHub Actions](https://github.com/features/actions). In your git repository, create and commit a file at `.github/workflows/deploy.yml`. Here is a starting example:
+Once the app has been set up, you can have Observable build the next deploys for you. You can opt in to build *on commit* (whenever you push a new commit to your project’s default branch), and *on schedule*, such as daily or weekly.
+
+Automatic deploys — also known as *continuous deployment* or *CD* — ensure that your  data is up to date, and that any changes you make to your app are immediately reflected in the deployed version.
+
+On your project’s configuration page on Observable, open the **Build settings** tab and click **Connect to GitHub**. This will prompt you to sign in to GitHub and authorize Observable to access (a selection of) your repositories. Once you’ve done that, you can select the project repository and branch you want to deploy from. Observable will then listen for changes in the repo and deploy the app automatically.
```

Comment:
i think those whole paragraph could be cut; you’ve covered all you need to when you say “on commit” above

Reply:
i’ve been saying “data app settings page” or just “app settings” 🤷